pilot/cmd/sidecar-injector/main.go

```diff
@@ -203,6 +205,8 @@ func init() {
 		"File containing the Istio mesh configuration")
 	rootCmd.PersistentFlags().StringVar(&flags.injectConfigFile, "injectConfig", "/etc/istio/inject/config",
 		"File containing the Istio sidecar injection configuration and template")
+	rootCmd.PersistentFlags().StringVar(&flags.injectValuesFile, "injectValues", "/etc/istio/inject/values",
+		"File containing the Istio sidecar injection values")
```

Comment:
It isn't clear if this file is json or yaml from the flags.  It may not make a difference in the implementation, but could you specify one or the other (or alter the description field to indicate json/yaml format?)